src/services/api/HttpService.ts

```diff
@@ -21,15 +21,13 @@ class HttpService {
     headerKeys.forEach((key) => delete this.client.defaults.headers[key]);
   }
 
-  //@ts-ignore
-  handleSuccessResponse(response) {
+  handleSuccessResponse(response: AxiosResponse) {
     return response;
   }
 
-  // @ts-ignore
-  handleErrorResponse = async (error) => {
+  handleErrorResponse = async (error: AxiosError) => {
     try {
-      const { status } = error.response;
+      const { status } = error.response!;
```

Comment:
I'm assuming this is typo
```suggestion
      const { status } = error.response;
```